Rubberduck.Parsing/ComReflection/LibraryReferencedDeclarationsCollector.cs

```diff
@@ -19,7 +20,7 @@ public override IReadOnlyCollection<Declaration> CollectedDeclarations(Reference
             return LoadDeclarationsFromLibrary(reference);
         }
 
-        private List<Declaration> LoadDeclarationsFromLibrary(ReferenceInfo reference)
+        private IReadOnlyCollection<Declaration> LoadDeclarationsFromLibrary(ReferenceInfo reference)
         {
             var libraryPath = reference.FullPath;
             // Failure to load might mean that it's a "normal" VBProject that will get parsed by us anyway.
```

Comment:
Does the comment need to be updated?

Reply:
The only difference would be changing _parsed_ to _loaded_.